### PR TITLE
Fix nested URLs do not work

### LINKED ISSUE #6: Nested URLs do not work.
Real URL: site.com/**_company_/about**
URL in sitemap.xml: site.com/**about**

### PR DESCRIPTION
**Pull request by Robot72**
Fix [Nested URLs do not work](https://github.com/cebe/luya-module-sitemap/issues/6)

### CODE REVIEW

**Comment by cebe** [comment]
Thank you for the contribution, could you also add a unit test case?

**Comment by Robot72** [comment]
> Thank you for the contribution, could you also add a unit test case?

No problem. I'll do it in the next few days.

**Comment by testt23** [comment]
@Robot72 can you help me with this problem, please. thank you

https://github.com/cebe/luya-module-sitemap/issues/8

File: src/controllers/SitemapController.php [hunk 1]
```diff
@@ -84,4 +91,41 @@ private function buildSitemapfile($sitemapFile)
...
+                'id' => $parentNavId,
+            ])->one();
+
+            $parentNavItem = $parentNav->getActiveLanguageItem()->one();
+            $alias = $parentNavItem->attributes['alias'];
+            $fullUriPath = $alias . '/' . $fullUriPath;
+            $parentNavId = $parentNav->attributes['parent_nav_id'];
+            $nestingIndex += 1;
+            if ($nestingIndex >= $this->maxPathNesting) {
+                // TODO create Exception or other action for notify to customers
```
Comment (cebe) [inline src/controllers/SitemapController.php:124]: why did you add this limit? the nesting is limited by data in CMS, I do not think adding a limit with arbitrary value here makes sense.

Comment (Robot72) [reply src/controllers/SitemapController.php:124]: I did not know CMS was somehow limited to nesting for URL path.

I did not find specific recommendations on the level of nesting but I have a few SEO experts said that it is better to make nesting no more than 10.

Comment (cebe) [reply src/controllers/SitemapController.php:124]: sure, that is about SEO and best practise, but there is no point in limiting it in the sitemap.xml.

**Comment by cebe** [comment]
Merged, thank you!

FYI: I made an additional change to use correct language in parent pages.

**Pull request merged**